### PR TITLE
Wrap Chains GQL queries on payloads

### PR DESCRIPTION
**Pull request by vyzaldysanchez**
Closes: https://app.shortcut.com/chainlinklabs/story/20931/wrap-chain-queries-mutations-in-payload

This aims to return proper payloads for all Chains related GQL queries.

### CODE REVIEW

File: core/web/resolver/query.go [hunk 1]
```diff
@@ -47,16 +47,16 @@ func (r *Resolver) Bridges(ctx context.Context, args struct {
 	offset := pageOffset(args.Offset)
 	limit := pageLimit(args.Limit)
 
-	bridges, count, err := r.App.BridgeORM().BridgeTypes(offset, limit)
+	btps, count, err := r.App.BridgeORM().BridgeTypes(offset, limit)
```
Comment (jkongie) [inline core/web/resolver/query.go:50]: `bridges` is ok here. BridgeType should really be renamed

**Review by jkongie — CHANGES_REQUESTED** [review]

**Review by jkongie — APPROVED** [review]

**Review by makramkd — APPROVED** [review]
:shipit:

**Comment by vyzaldysanchez** [comment]
@cl-homu r+

**Comment by cl-homu** [comment]
:pushpin: Commit b96617088f9683e8d916ec65b031e5d2a4650e8b has been approved by `vyzaldysanchez`

**Comment by cl-homu** [comment]
:hourglass: Testing commit b96617088f9683e8d916ec65b031e5d2a4650e8b with merge 5458209f2f45a359bd1830fd562d16818415b76f...

**Comment by cl-homu** [comment]
:sunny: Test successful - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4183350071?check_suite_focus=true), [checks-lint](https://github.com/smartcontractkit/chainlink/runs/4183350001?check_suite_focus=true), [checks-npm](https://github.com/smartcontractkit/chainlink/runs/4183349916?check_suite_focus=true), [checks-operator](https://github.com/smartcontractkit/chainlink/runs/4183349996?check_suite_focus=true), [checks-prettier](https://github.com/smartcontractkit/chainlink/runs/4183350108?check_suite_focus=true), [checks-solidity](https://github.com/smartcontractkit/chainlink/runs/4183350077?check_suite_focus=true)
Approved by: vyzaldysanchez
Pushing 5458209f2f45a359bd1830fd562d16818415b76f to develop...

**Pull request merged**